ui/app/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -117,74 +123,95 @@ export default class SelectHardware extends Component {
     );
   }
 
-  getAffiliateLinks() {
-    const links = {
-      trezor: `<a class='hw-connect__get-hw__link' href='https://shop.trezor.io/?a=metamask' target='_blank'>Trezor</a>`,
-      ledger: `<a class='hw-connect__get-hw__link' href='https://www.ledger.com/products/ledger-nano-s?r=17c4991a03fa&tracker=MY_TRACKER' target='_blank'>Ledger</a>`,
-    };
-
-    const text = this.context.t('orderOneHere');
-    const response = text
-      .replace('Trezor', links.trezor)
-      .replace('Ledger', links.ledger);
-
-    return (
-      <div
-        className="hw-connect__get-hw__msg"
-        dangerouslySetInnerHTML={{ __html: response }}
-      />
-    );
-  }
-
-  renderTrezorAffiliateLink() {
-    return (
-      <div className="hw-connect__get-hw">
-        <p className="hw-connect__get-hw__msg">
-          {this.context.t('dontHaveAHardwareWallet')}
-        </p>
-        {this.getAffiliateLinks()}
-      </div>
-    );
+  renderTutorialsteps() {
+    switch (this.state.selectedDevice) {
+      case 'ledger':
+        return this.renderLedgerTutorialSteps();
+      case 'trezor':
+        return this.renderTrezorTutorialSteps();
+      default:
+        return '';
+    }
   }
 
-  scrollToTutorial = () => {
-    if (this.referenceNode) {
-      this.referenceNode.scrollIntoView({ behavior: 'smooth' });
+  renderLedgerTutorialSteps() {
+    const steps = [];
+    if (this.props.useLedgerLive) {
+      steps.push({
+        title: this.context.t('step1LedgerWallet'),
+        message: this.context.t('step1LedgerWalletMsg', [
+          // eslint-disable-next-line react/jsx-key
+          <a
+            className="hw-connect__msg__link"
+            href="https://www.ledger.com/ledger-live"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Ledger Live App
+          </a>,
+        ]),
+      });
     }
-  };
 
-  renderLearnMore() {
+    steps.push({
+      asset: 'plug-in-wallet',
+      dimensions: { width: '225px', height: '75px' },
+      title: this.context.t('step2LedgerWallet'),
+      message: this.context.t('step2LedgerWalletMsg', [
+        // eslint-disable-next-line react/jsx-key
+        <a
+          className="hw-connect__msg__link"
+          href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          {this.context.t('hardwareWalletSupportLinkConversion')}
+        </a>,
+      ]),
+    });
+
     return (
-      <p className="hw-connect__learn-more" onClick={this.scrollToTutorial}>
-        {this.context.t('learnMore')}
-        <img
-          className="hw-connect__learn-more__arrow"
-          src="images/caret-right.svg"
-          alt=""
-        />
-      </p>
+      <div
+        className="hw-tutorial"
+        ref={(node) => {
+          this.referenceNode = node;
```

Comment:
This ref was only set for the purposes of auto-scrolling, which was removed (see the removed `scrollToTutorial` function).  This can be removed as well. The same goes for the second copy of this below as well.

Reply:
++